spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/source/TestMetadataTablesWithPartitionEvolution.java

```diff
@@ -212,7 +214,6 @@ public void testFilesMetadataTableFilter() throws ParseException {
     sql("CREATE TABLE %s (id bigint NOT NULL, category string, data string) USING iceberg " +
         "TBLPROPERTIES ('commit.manifest-merge.enabled' 'false')", tableName);
     initTable();
-
```

Comment:
Unnecessary white space change here

Reply:
I think if we want to make this change we need to do it in all the tests and currently this is missing format changes for testFilesMetadataTable and testWithUnknownTransfer. Probably fine to just keep it as is and match in the new tests

Reply:
Removed whitespace